packages/redis/src/lib/subscription-manager.ts

```diff
@@ -0,0 +1,83 @@
+import { randomUUID } from "crypto";
+
+import type { MaybePromise } from "@homarr/common/types";
+import { logger } from "@homarr/log";
+
+import { createRedisConnection } from "./connection";
+
+type SubscriptionCallback = (message: string) => MaybePromise<void>;
+
+export class SubscriptionManager {
```

Comment:
``Manager`` is a poorly chosen name

Reply:
What do you think of `ChannelSubscriptionTracker`?
If not any better ideas?

Reply:
lgtm 👍